9.2/root/usr/bin/hooks.d/post-init

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
@mnagy Not sure I follow -- for which step you would like to use those variables? For running the SQL queries in post-init?